.rubocop.yml

```diff
@@ -832,6 +832,10 @@ Rails/Inquiry:
   StyleGuide: 'https://rails.rubystyle.guide/#inquiry'
   Enabled: true
 
+Rails/LexicallyScopedActionFilter:
+  Exclude:
+    - app/controllers/api/**/*
+
```

Comment:
Rubocop doesn't like `before_action` referencing actions defined in a different file. This should be removed as soon as we finish the transition period from `V0` to `V1`

Reply:
Me too Rubocop…me too.  But I agree that for this transition period this is an adequate approach.  Would inline disables be more appropriate? Perhaps. But not so much as to require re-work